shared/src/license.ts

```diff
@@ -9,7 +9,9 @@ export type LicenseStatusResult = {
   checklistItems: LicenseStatusItem[];
 };
 
-export type CheckoffStatus = "ACTIVE" | "PENDING" | "UNKNOWN";
+export type CheckoffStatus = "ACTIVE" | "PENDING" | "INCOMPLETE" | "SCHEDULED" | "NOT_APPLICABLE" | "UNKNOWN";
+
+export type ChecklistStatusText = "Completed" | "Incomplete" | "Pending" | "Scheduled" | "N/A";
```

Comment:
[dust] thoughts on moving this into the `ChecklistTag.tsx` file as it's only used there currently and it isn't necessarily shared?